resources/zib/zib-MedicationDispense.xml

```diff
@@ -149,6 +150,23 @@
         <comment value="Supplier" />
       </mapping>
     </element>
+    <element id="MedicationDispense.location">
+      <path value="MedicationDispense.location" />
+      <short value="Supplier" />
+      <definition value="In almost all cases, the supplier will be a pharmacist. It could also be supplied by a webshop (in case of an online order), a drug store or a foreign pharmacist." />

```

Comment:
Staat dit letterlijk zo in zib/MP dataset? Want bij een 'webshop' moet natuurlijk ook een apotheker werken als het gaat om geneesmiddelen op recept ? Bij over the counter (zoals ook drogisterij) geldt dat natuurlijk niet. Een webshop is waarschijnlijk (net als een apotheek) een beetje van beide.

Reply:
Dit staat letterlijk zo in de webshop